source/asyncify.d.ts

```diff
@@ -0,0 +1,27 @@
+import {PromiseValue} from './promise-value';
+import {SetReturnType} from './set-return-type';
+
+/**
+Create an async version of the given function type.
+
+@example
+```ts
```

Comment:
```suggestion
```
```

Reply:
You probably mean to delete just the `ts`, not the whole thing, right?